scripts/rosws.bat

```diff
@@ -1,2 +1,2 @@
-cd %USERPROFILE%\SYSC4907-AirSim\SAVE-master\catkin_ws
```

Comment:
Can you not push this? Enables new users to run the scripts if they load the repo into their user folder.

Reply:
I have no idea how that file got updated. Should be reverted now